README.md

```diff
@@ -1,2 +1,18 @@
-# baips
-BitAssets Improvement Proposals
+# BitAssets Improvement Proposal
+
+BAIP stands for BitAssets Improvement Proposal but can also seen as an
+improvement *consensus*. BitAssets are BitShares Committee's SmartCoins
```

Comment:
`Committee's SmartCoins` doesn't sound accurate.

Reply:
The BAIP description in forum sound better. 
https://bitsharestalk.org/index.php?topic=29723